src/styles/header.scss

```diff
@@ -0,0 +1,85 @@
+// #region header
+.header {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.navi-bar {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.navi-menu {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.navi-item {
+  margin: 0;
+  padding-left: 20px;
+}
+
+.navi-item:first-child {
+  padding: 0;
+}
+
+.navi-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  color: #060b35;
+  height: 100%;
+  line-height: 60px;
+  padding: 0;
+  text-align: left;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.navi-link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  display: flex;
+
+  text-decoration: none;
+  color: #00acdc;
+  line-height: 60px;
+  padding: 0;
+  text-align: left;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  margin-left: 0;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
```

Comment:
The `::after` pseudo-element is well used here to create the underline for the active link. However, the `border-radius: 8px;` seems unnecessary because the height of the element is only 4px, so it won't be visible.